pkgs/by-name/gh/gh/package.nix

```diff
@@ -23,9 +23,18 @@ buildGoModule rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
+  ldflags = [
```

Comment:
It is actually confusing to use `ldflags` as they only work for packages using the default build/install phase.

Please just add `-X github.com/cli/cli/v2/internal/build.Date=nixpkgs"` to the existing GO_LDFLAGS in the build phase.

Reply:
OK, I was hoping it would make it more readable. I'll make the change